snippets/javascript/nuxt/nuxt.json

```diff
@@ -0,0 +1,87 @@
+{
+  "nuxt-nuxtlink": {
+    "prefix": ["nuxt nuxtlink", "nuxtlink"],
+    "body": ["<nuxt-link $1>$2</nuxt-link>$0"],
+    "description": "nuxt-link element",
+    "scope": "vue-html"
+  },
+  "nuxt-nuxtlinkto": {
+    "prefix": ["nuxt nuxtlinkto", "nuxtlinkto"],
+    "body": ["<nuxt-link to=\"$1\">$2</nuxt-link>$0"],
+    "description": "<nuxt-link to=''></nuxt-link>. nuxt-link element",
+    "scope": "vue-html"
+  },
+  "nuxt-nuxtnamedlinkto": {
+    "prefix": ["nuxt nuxtnamedlinkto", "nuxtnamedlinkto"],
+    "body": [
+      "<nuxt-link :to=\"{\t${1:key}: '${2:value}', ${3:key}: {\t${4:key}: '${5:value}'\t}\t}\">$6</nuxt-link>$0"
```

Comment:
I've read briefly nuxt docs, am I right that better will be to change it to:
`<nuxt-link :to=\"{\t${1:name}: '${2:value}', ${3:params}: {\t${4:key}: '${5:value}'\t}\t}\">$6</nuxt-link>$0`

Reply:
Okay. I left it as ${*:key} because it could be name or path for 1 and params or query for 3. You can check here. https://router.vuejs.org/api/#to